pages/mfbarrera/style.css

```diff
@@ -0,0 +1,141 @@
+body {
+    margin: 0;
+    padding: 0;
+    width: 1480px;
+}    
+
+.background{
+    background: linear-gradient(0deg, #1d1d1d9f, #313131ab), url(https://revistaroomin.com/wp-content/uploads/2019/03/travel-editor-favorite-products.jpg);
+    background-size: cover;
+    height: 600px;
+    position: absolute;
+    width: 90%;
+}    
+
+.button{
+    background: transparent;
+    border: none;
+    height: 40px;
+    width: 40px;
+    padding: 0; 
+    position: absolute;
+    top: 50%;
+} 
+
+.button--right {
+    right: 5%;
+    transform: rotate(180deg);
+}
+
+.button--left {
+    left: 5%;
+}
+
+.button__image {
+    width: 100%;
+    height: 100%;
+}   
+
+.card {
+    background-color: rgb(255, 255, 255); 
+    width: 180px;
+    margin: 20px 16px ;
+    text-align: center;
+    padding: 16px;
+    display: flex;
+    flex-direction: column;
+    border-radius: 5px;
+    height: 300px;
+    justify-content: space-between;   
+}    
+
+.card:hover{
```

Comment:
This animation is not required yet.  Please remember two key concepts:  Correctness and completeness.

Reply:
it is a new feature for my cards